extensions/applicationinsights-react-native/src/ReactNativePlugin.ts

```diff
@@ -12,15 +12,33 @@ import {
     LoggingSeverity,
     _InternalMessageId,
     BaseTelemetryPlugin,
-    IProcessTelemetryContext,
-    objForEachKey
+    IProcessTelemetryContext//,
+    // objForEachKey
 } from '@microsoft/applicationinsights-core-js';
 import { ConfigurationManager, IDevice, IExceptionTelemetry, IAppInsights, SeverityLevel, AnalyticsPluginIdentifier  } from '@microsoft/applicationinsights-common';
 import DeviceInfo from 'react-native-device-info';
 
 import { INativeDevice, IReactNativePluginConfig } from './Interfaces';
 import dynamicProto from '@microsoft/dynamicproto-js';
 
+
+/**
```

Comment:
??

Reply:
And generally, the release PR should only be bumping the numbers and not include and code changes